.github/workflows/test_pull_requests.yml

```diff
@@ -107,6 +111,11 @@ jobs:
             backend: pyqt5
             MIN_REQ: 1
             coverage: cov
+          - python: "3.10"
+            platform: ubuntu-22.04
+            backend: pyqt5
+            coverage: cov
+            pydantic: ""
           # test without any Qt backends
```

Comment:
Not sure I follow the changes to the workflows. So there's one extra env in the matrix which tests pydantic>=2 with python 3.10, and the rest is same as before?

Reply:
Yes.

Reply:
Got it!